management-app/src/main/java/com/nearsoft/fgaribay/mgmt/config/BrokerConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.nearsoft.fgaribay.mgmt.config;
+
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.QueueBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class BrokerConfig {
+    @Value("product.create.queue.name")
```

Comment:
Explore the usage of `@ConfigurationProperties` to get this values from your settings.
- [Type-safe Configuration Properties](https://docs.spring.io/spring-boot/docs/1.5.7.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties)